frontstage/common/message_helper.py

```diff
@@ -81,3 +82,16 @@ def get_formatted_date(datetime_string, string_format='%Y-%m-%d %H:%M:%S'):
     elif time_difference.days == -1:
         return "Yesterday at {}".format(datetime_parsed.strftime('%H:%M'))
     return "{} {}".format(datetime_parsed.strftime('%d %b %Y').title(), datetime_parsed.strftime('%H:%M'))
+
+
+def get_current_date_bst(datetime_string):
```

Comment:
Only thing with this is that it will work for respondents in the uk, but not in other places with different timezones

Reply:
good point, could add an if statement to determine locale, where this is called, not sure on the requirement though. Ticket only stated that the displayed time should be BST